deployment/docker-compose/litellm_proxy_config.yaml

```diff
@@ -3,11 +3,11 @@ model_list:
     litellm_params:
       model: text-embedding-3-small
       api_key: "os.environ/OPENAI_API_KEY"
-  - model_name: embeddings-local
-    litellm_params:
-      model: openai/embedding-gte-large
-      api_key: "os.environ/EMBEDDINGS_API_KEY"
-      api_base: "os.environ/EMBEDDINGS_ENDPOINT"
+  # - model_name: embeddings
+  #   litellm_params:
+  #     model: huggingface/Alibaba-NLP/gte-large-en-v1.5
+  #     api_key: "os.environ/HUGGINGFACE_API_KEY" #pragma: allowlist secret
+  #     api_base: "os.environ/EMBEDDINGS_ENDPOINT"
```

Comment:
Can you add the following to the dev setup docs that if you want to use local HF embeddings you need to:

1. Set HUGGINGFACE_MODEL and HUGGINGFACE_API_KEY and run either of the make commands for spinning up the embeddings containers
2. Also set EMBEDDINGS_ENDPOINT and uncomment this section and comment out the openai embeddings model
3. finally run `make setup-dev`

The more detail the better. I might be missing smt.